tools/runners/test262-harness.py

```diff
@@ -708,9 +785,15 @@ def percent_format(partial, total):
 
 class TestSuite(object):
 
+    #pylint: disable=too-many-instance-attributes
     def __init__(self, options):
-        self.test_root = path.join(options.tests, 'test')
-        self.lib_root = path.join(options.tests, 'harness')
+        self.es5_1 = options.es5_1
+        if self.es5_1:
+            self.test_root = path.join(options.tests, 'test', "suite")
+            self.lib_root = path.join(options.tests, 'test', 'harness')
+        else:
+            self.test_root = path.join(options.tests, 'test')
+            self.lib_root = path.join(options.tests, 'harness')
```

Comment:
```py
test_folder = '' if self.es5_1 else 'test';
self.test_root = path.join(options.tests, test_folder, 'test')
self.lib_root = path.join(options.tests, test_folder, 'harness')

```

Reply:
> ```python
> test_folder = '' if self.es5_1 else 'test';
> self.test_root = path.join(options.tests, test_folder, 'test')
> self.lib_root = path.join(options.tests, test_folder, 'harness')
> ```

Not what you think, it's 'suite'
            self.test_root = path.join(options.tests, 'test', "suite")

Reply:
Indeed, sorry I've missed that.